File: server/Models/SupportTickets.cs
namespace PCpals.Models;

public class SupportTickets : RepoItem<int>{
    public string CreatorId {get; set;}
    public string CustomerName {get; set;}
    public string CustomerEmail {get; set;}
    public string IssueSubject {get; set;}
    public string IssueDescription {get; set;}
    public string TicketStatus {get; set;}
    public Account Creator {get; set;}
}

File: client/src/models/SupportTickets.js
export class SupportTickets{
    constructor(data){
        this.id = data.id
        this.creatorId = data.creatorId
        this.customerName = data.customerName
        this.customerEmail = data.customerEmail
        this.issueSubject = data.issueSubject
        this.issueDescription = data.issueDescription
        this.ticketStatus = data.ticketStatus
        this.creator = data.creator
    }
}
